modules/services/cloud-connector-ecs/versions.tf

```diff
@@ -1,6 +1,5 @@
 terraform {
-  required_version = ">= 1.0.0"
-  experiments      = [module_variable_optional_attrs]
+  required_version = ">= 1.3.0"
```

Comment:
Does it mean we are dropping support for Terraform < 1.3.0? This might be a concern for some customers, but I guess there is no way to conditionally check the version as part of the HCL code.

Reply:
1.3 is breaking change, and will not work with experimental flag on.
if some customer needs < 1.3 they will be required to pin down specific SFC module version. 

gonna open another PR indeed to avoid confusion and be able to clarify this publicly 🤷🏽‍♀️ 👍🏽

Reply:
https://github.com/sysdiglabs/terraform-aws-secure-for-cloud/pull/132